### ISSUE TITLE
[Feature Request] STAC algorithm

### ISSUE THREAD

**Issue by EloyAnguiano** [Stable-Baselines-Team/stable-baselines3-contrib#223]
### 🚀 Feature

Build the STAC algorithm as a callable algorithm: https://arxiv.org/pdf/2002.12928.pdf

### Motivation

Hyperparametrization is one of the most time/cost expensive thing when training RL agents. May be this implementation saves some time/cost to some people and it could be the first AC algorithms that deals with meta-gradients to make improvements from here.

### Pitch

I would like some to guide me of where to start or to give me some key insights of the posibilities of coding this.

### Alternatives

The alternatives are that someone codes it by him/herself. 

### Checklist

- [X] I have checked that there is no similar [issue](https://github.com/Stable-Baselines-Team/stable-baselines3-contrib/issues) in the repo
- [X] If I'm requesting a new feature, I have proposed alternatives

**Comment by araffin** [comment]
Hello,
are you willing to implement and benchmark the algorithm?

**Comment by EloyAnguiano** [comment]
The algorithm is an Off-policy one. Is there any way or example to begin with this kind of algorithms?

**Comment by araffin** [comment]
> The algorithm is an Off-policy one. Is there any way or example to begin with this kind of algorithms?

https://github.com/Stable-Baselines-Team/stable-baselines3-contrib/pull/4
and
please read https://github.com/Stable-Baselines-Team/stable-baselines3-contrib/blob/master/CONTRIBUTING.md
